launcher/minecraft/launch/LauncherPartLaunch.cpp

```diff
@@ -159,6 +160,74 @@ void LauncherPartLaunch::executeTask()
 
     qDebug() << args.join(' ');
 
+    args.prepend(FS::ResolveExecutable(instance->settings()->get("JavaPath").toString()));
+
+#ifdef Q_OS_LINUX
+    // TODO: Sandboxing setting
+    if (true) {
+        QString actualRuntimeDir = QStandardPaths::writableLocation(QStandardPaths::RuntimeLocation);
+        QString sandboxedRuntimeDir = "/tmp";
+
+        QStringList bwrapArgs{};
+        bwrapArgs << QStandardPaths::findExecutable(BuildConfig.LINUX_BWRAP_BINARY);
+        bwrapArgs << "--unshare-user" << "--unshare-ipc" << "--unshare-pid" << "--unshare-uts" << "--unshare-cgroup";
+        bwrapArgs << "--share-net";
+        bwrapArgs << "--die-with-parent";
+        bwrapArgs << "--unsetenv" << "DBUS_SESSION_BUS_ADDRESS";
+
+        // default binds
+        bwrapArgs << "--dev" << "/dev";
+        bwrapArgs << "--dev-bind-try" << "/dev/dri" << "/dev/dri";
+        bwrapArgs << "--proc" << "/proc";
+        bwrapArgs << "--tmpfs" << "/tmp";
```

Comment:
Root itself is already a tmpfs. Is `/tmp` required to be a separated mount point?